web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElement.test.tsx

```diff
@@ -65,6 +67,14 @@ describe("<AnyTimeActionTaxClearanceCertificateReviewElement />", () => {
     );
   };
 
+  const generateBusinessWithEmptyTaxClearanceData = (): Business => {
+    return generateBusiness({
+      profileData: generateProfileData({ businessName: "", taxId: undefined, taxPin: undefined }),
+      taxClearanceCertificateData: emptyTaxClearanceCertificateData,
+      formationData: generateFormationData({ formationFormData: createEmptyFormationFormData() }),
```

Comment:
you should remove formationData: generateFormationData({ formationFormData: createEmptyFormationFormData() }),` from your staged data, the behavior does not rely on formation data

Reply:
Because the tax clearance data is initial seeded by any values that might be present in the formation address or profile, I think this is here to just doubly ensure that's those fields are empty. When I remove this I'm not seeing any tests fail, but I'm going to leave it in unless there's a strong reason I'm not considering.